geogig/src/main/java/org/geogig/geoserver/config/GeoServerStoreRepositoryResolver.java

```diff
@@ -18,8 +18,8 @@ public URI resolve(final String repository) {
             //enforce unix file seperators
             String unixsep = FilenameUtils.separatorsToUnix(info.getLocation());
             //check if refers to file, if it does add the file prefix
-            if(FilenameUtils.getPrefix(unixsep)!=null&&!unixsep.startsWith("file:/"))
-            	unixsep = "file:/"+unixsep;
+            if(FilenameUtils.getPrefix(unixsep)!=null&&!unixsep.startsWith("file://"))
```

Comment:
@jmiller-boundless Why don't we do this:
```
File locFile = new File(info.getLocation());
return locFile.toUri();
```
Java's File object should return the correct "file:" URI for the location, regardless of platform. I can try it out on Linux, but I don't have a Windows platform I can test it on. Can you let me know if it works on Windows?

Looking at the "getLocation()" method, it is returning the getAbsolutePath() method on File, so that should be platform specific, and should yield a String that can be used to construct a File object that we can call toUri() on, without having to do all the platform checks.